Bundle/app/src/main/java/com/scottlindley/Bundle/DetailView/SettingsFragment.java

```diff
@@ -0,0 +1,38 @@
+package com.scottlindley.Bundle.DetailView;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.scottlindley.Bundle.R;
+
+public class SettingsFragment extends Fragment {
```

Comment:
Since this class didn't end up functioning, why was it merged into your master branch?